docs/4.0/examples/floating-labels/floating-labels.css

```diff
@@ -82,3 +82,25 @@ body {
   font-size: 12px;
   color: #777;
 }
+
+/* Fallback for Edge
+-------------------------------------------------- */
+@supports (-ms-ime-align: auto) {
+  .form-label-group > label {
+    display: none;
+  }
+  .form-label-group input::-ms-input-placeholder {
+    color: #777 !important;
```

Comment:
Why is `!important` needed? Isn't this pseudo selector IE only?

This is the reason tests fail BTW.